hw/ip/flash_ctrl/dv/cov/flash_ctrl_cov.vRefine

```diff
@@ -0,0 +1,59 @@
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_i.a_user.cmd_intg&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_i.a_user.data_intg&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_i.d_ready&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_valid&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_opcode&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_param&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_size&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_source&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_sink&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_data&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_user.rsp_intg&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_user.data_intg&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.d_error&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
+    <rule ccType="inst" domain="icc" entityName="flash_ctrl/&quot;prim_tl_o.a_ready&quot;" entityType="toggle" excTime="1652804137" name="exclude" reviewer="unknown" user="0" vscope="default"></rule>
```

Comment:
@tjaychen looks like these are to exclude some toggle coverage for prim_generic_flash and prim_tl_*. can you confirm if these are correct?

Reply:
this will change very shortly. We are going to switch the alias register scheme, and these registers should then fall under normal csr_rw for a separate tlul interface. 

Wouldn't those be covered then?

Reply:
Here I excluded not used interface  prim_tl_*, like discussed in #12325 .  not registers. This interface is hanging in the air.

Reply:
core_tl and mem_tl have interfaces are connected and exercised via agents in TB but prim_tl is unused interface and therefore it needs to be excluded. Here is the picture of coverage:

![prim_tl_if](https://user-images.githubusercontent.com/93394800/170492431-a692a583-2a4f-4e0e-8949-39aba3f7a380.JPG)